packages/cloudflare/src/cli/build/patches/to-investigate/patch-find-dir.ts

```diff
@@ -11,18 +11,24 @@ import { Config } from "../../../config";
  */
 export function patchFindDir(code: string, config: Config): string {
   console.log("# patchFindDir");
-  return code.replace(
-    "function findDir(dir, name) {",
-    `function findDir(dir, name) {
-			if (dir.endsWith(".next/server")) {
-			if (name === "app") {
+  const patchedCode = code.replace(
```

Comment:
Maybe add a comment here with the pre-15.1 code and 15.1 code

Reply:
are you sure? it feels a bit overkill to me here 😕

Reply:
It might help if we later need to convert the patch to AST but your call :)

Reply:
I'd leave it as is if it's ok by you 🙂 

but if you do prefer a comment, could you please add a code suggestion here and I'd be happy to just go with whatever you'd fine useful here 🙂